specification.markdown

```diff
@@ -386,14 +386,14 @@ The path defines the allowed storage locations of the EBML Element within an EBM
 The path attribute is REQUIRED.
 
 ```
-EBMLFullPath          = [EBMLParentPath] EBMLElementPath
-EBMLParentPath        = EBMLFixedParent EBMLLastParent
-EBMLFixedParent       = *(EBMLPathAtom)
-EBMLElementPath       = EBMLPathAtom / EBMLPathAtomRecursive
+EBMLFullPath          = [EBMLParentPath] [GlobalPlaceholder] 
+                        EBMLLastPath
+EBMLLastPath          = EBMLPathAtom / EBMLPathAtomRecursive
+EBMLParentPath        = 1*(IntermediatePathAtom)
+IntermediatePathAtom  = EBMLPathAtom / EBMLPathAtomRecursive
+EBMLPathAtomRecursive = "(" EBMLPathAtom ")"
 EBMLPathAtom          = PathDelimiter EBMLAtomName
-EBMLPathAtomRecursive = "(1*(" EBMLPathAtom "))"
-EBMLLastParent        = EBMLPathAtom / EBMLGlobalParent
-EBMLGlobalParent      = "(" GlobalParentOccurence "\)"
+GlobalPlaceholder     = "(" GlobalParentOccurence "\)"
 EBMLAtomName          = 1*(EBMLNameChar)
 EBMLNameChar          = ALPHA / DIGIT / "-" / "."
```

Comment:
EBMLNameChar doesn't match the spec. I think it should be something like
`EBMLNameChar = ALPHA / DIGIT 0*[ALPHA / DIGIT / "-" / "."]`

Reply:
Actually the `EBMLAtomName` start with be a `ALPHA / DIGIT`. `EBMLNameChar` is the list of possible chars after the first one. But good catch.